### ISSUE TITLE
JuliaFormatter 1.0

### ISSUE THREAD

**Issue by odow** [jump-dev/JuMP.jl#2990]
JuliaFormatter went 1.0, which should simplify our formatting actions: https://github.com/domluna/JuliaFormatter.jl/releases/tag/v1.0.0

There are a couple of options we could consider turning on:
 * https://github.com/domluna/JuliaFormatter.jl#separate_kwargs_with_semicolon
   Because it actually makes a difference in places:
   ```Julia
   julia> sum(i for i in 1:0; init = 0)
   0
   
   julia> sum(i for i in 1:0, init = 0)
   ERROR: ArgumentError: reducing over an empty collection is not allowed
   Stacktrace:
   ```
 * https://github.com/domluna/JuliaFormatter.jl#format_docstrings: blocked by https://github.com/domluna/JuliaFormatter.jl/issues/597

**Comment by odow** [comment]
Currently blocked by https://github.com/domluna/JuliaFormatter.jl/issues/603 and https://github.com/MichaelHatherly/CommonMark.jl/issues/43.

**Comment by odow** [comment]
Closing since we've updated to JuliaFormatter v1.0. Formatting the docstrings is a low priority, it's blocked upstream with no ETA, and no one has ever requested it.
